src/som/interpreter/actors/EventualSendNode.java

```diff
@@ -162,6 +162,8 @@ protected void sendDirectMessage(final Object[] args, final Actor owner,
       DirectMessage msg = new DirectMessage(
           EventualMessage.getCurrentExecutingMessage(), target, selector, args,
           owner, resolver, onReceive);
+
+      checkMessageBreakpointed(msg);
```

Comment:
I think, this should be a function that takes the target actor, and returns a boolean for whether or not the message should trigger a breakpoint.